### ISSUE TITLE
Button labels unreadable

### ISSUE THREAD

**Issue by SanderBouwhuis** [witalihirsch/qBitTorrent-fluent-theme#61]
I am trying out fluent-dark-no-mica.qbtheme, and it looks great!
However, there is a bug in the columns labels. They are unreadable (maybe because they are left aligned?!?):

![Labels bug](https://github.com/user-attachments/assets/c87392f5-ae3a-4041-a5d8-5381481f178b)

**Comment by froggz19** [comment]
Just resize the column?
![qbcolumnresize](https://github.com/user-attachments/assets/9fc0e724-9b8e-4c3c-b2d2-faee84c21e8c)

**Comment by SanderBouwhuis** [comment]
No, you can see in your own animation that half the label space is blank. The word 'progress' easily fits, but half the text disappears.
I hope this can be fixed, because I kinda like this theme.
